apisix/core/config_etcd.lua

```diff
@@ -181,15 +181,15 @@ local function load_full_data(self, dir_res, headers)
             data_valid, err = check_schema(self.item_schema, item.value)
             if not data_valid then
                 log.error("failed to check item data of [", self.key,
-                          "] err:", err, " ,val: ", json.encode(item.value))
+                          "] err:", err, ", val: ", json.delay_encode(item.value))
```

Comment:
We don't delay encode for error level log, which is high enough.

Reply:
okay, just format log output.